test/1.2/misc/porting.dml

```diff
@@ -428,4 +430,9 @@ method init() {
             $c.signal.signal_raise();
         }
     }
+    if (1) {
+        assert $zero;
```

Comment:
Since `PZEROCOND` covers both `assert 0;` and `assert 1;`, test `assert 1;` too. Also, because of that, you may want to consider renaming it to `PILITCOND` -- as in, Integer Literal Condition.